src/utils/googleTagManager.ts

```diff
@@ -76,21 +80,17 @@ export const loadGoogleTagManager = (): void => {
   // Cache name to prevent tracking of same page
   currentPathname = history.location.pathname
 
-  const page = getAnonymizedLocation()
-
+  const page_path = getAnonymizedPathname()
   TagManager.initialize({
     gtmId: GOOGLE_TAG_MANAGER_ID,
     ...GTM_ENVIRONMENT,
     dataLayer: {
       // Must emit (custom) event in order to trigger page tracking
       event: GTM_EVENT.PAGEVIEW,
       chainId: _getChainId(),
-      page,
-      // Allow only GA4 configuration and GA4 custom event tags
-      // @see https://developers.google.com/tag-platform/tag-manager/web/restrict
-      'gtm.allowlist': ['gaawc', 'gaawe'],
-      // Block JS variables and custom scripts
-      'gtm.blocklist': ['j', 'jsm', 'customScripts'],
+      pageLocation: `${location.origin}${page_path}`,
+      pagePath: page_path,
```

Comment:
These are custom `dataLayer` variables that are mapped (in GTM) to overwrite GAs `page_location`, `page_path` and `page_referrer`

Reply:
Does this overwrite all events 'fired', i.e. `user_engagement`, `first_visit` and `session_start`? Those seem to be the 'offending' events.

Very interesting that the keys are camel case.